frontend/app/src/people/widgetViews/ManageOrgButton.tsx

```diff
@@ -0,0 +1,47 @@
+import { Button } from 'components/common';
+import { userHasRole } from 'helpers';
+import React, { useState, useEffect, useCallback } from 'react';
+import { useStores } from 'store';
+
+const ManageButton = (props: { user_pubkey: string, org: any, action: () => void }) => {
+    const [userRoles, setUserRoles] = useState<any[]>([]);
+    const { main, ui } = useStores();
+
+    const { user_pubkey, org, action } = props;
+
+    const getUserRoles = useCallback(async () => {
+        const userRoles = await main.getUserRoles(org.uuid, user_pubkey);
+        setUserRoles(userRoles);
+    }, [org.uuid])
+
+    useEffect(() => {
+        getUserRoles();
+    }, [getUserRoles])
+
+
+    const isOrganizationAdmin = org?.owner_pubkey === ui.meInfo?.owner_pubkey;
+
+    return (
+        <>
+            {
+                (isOrganizationAdmin ||
+                    userHasRole(main.bountyRoles, userRoles, 'ADD USER') ||
+                    userHasRole(main.bountyRoles, userRoles, 'VIEW REPORT'))
```

Comment:
lets just define this as a boolean above to make the react part look simple, we can call it `shouldShowButton: boolean`